File: MapAccounts/MapAccounts/Client/Scripts/Primitives/FilterResultDTO.js
function FilterResultDTO()
{
    this.imageID = -1;
    this.base64image = "";
    //PointDTO
    this.location = {};
    /*
    public enum CaracteristicType
        {
            Trees,
            Cracks

        }
    */
    this.type = null;
    this.isCaracteristicPresent = null;
    this.density = null;
    this.processedArea = null;
}

FilterResultDTO.prototype.initialize = function (_imageID, _base64image, _location, _type, _isCaracteristicPresent, _density, _processedArea)
{
    this.imageID = _imageID;
    this.base64image = _base64image;
    this.location = _location;
    this.type = _type;
    this.isCaracteristicPresent = _isCaracteristicPresent;
    this.density = _density;
    this.processedArea = _processedArea;
}

File: MapAccounts/MapAccounts/Models/Primitives/FilterResultDTO.cs
using System;

namespace MapAccounts.Models.Primitives
{
    public class FilterResultDTO
    {
        public FilterResultDTO()
        {
            this.imageID = -1;
            this.panoID = null;
        }

        public enum CaracteristicType
        {
            Trees,
            Cracks,
            Generic

        }
        //public PointDTO Location { get; set; }
        public int imageID { get; set; }
        public String panoID { get; set; }
        public CaracteristicType Type { get; set; }
        public String base64image { get; set; }
        public Boolean IsCaracteristicPresent { get; set; }
        public Double? Density { get; set; }
        public int ProcessedArea { get; set; }
    }
}